tests/Silverback.Integration.Tests.E2E/Kafka/CallbacksTests.cs

```diff
@@ -151,6 +156,177 @@ public async Task EndpointsConfiguredCallback_MessagePublished()
             kafkaTestingHelper.Spy.OutboundEnvelopes.Should().HaveCount(1);
         }
 
+        [Theory]
+        [InlineData(true, 6)] // There are 5 partitions and one message will be published, so in fact 6 times the end of the partition will be reached
```

Comment:
If you assume there are 5 partition, it would be nice to explicitly configure 5 in the test instead of relying on the default. Just for readability.